src/components/DateInput/styles.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+import { Open_Sans as OpenSans } from 'next/font/google';
+import COLORS from '@/styles/colors';
+import { H4 } from '@/styles/text';
+
+const openSans = OpenSans({ subsets: ['latin'] });
+
+const getBorderColor = (error: boolean, filled: boolean): string => {
+  if (error) {
+    return COLORS.redMid;
+  }
+  if (filled) {
+    return COLORS.greyMid;
+  }
+  return COLORS.greyLight;
```

Comment:
nit: we can turn these if statements to be one liners without the curly braces (i.e. `if (error) return COLORS.redMid`), could also consolidate the second if statement to a ternary statement (i.e. `return (filled) ? COLORS.greyMid : COLORS.greyLight`)